src/main/scala/higherkindness/rules_scala/workers/deps/DepsRunner.scala

```diff
@@ -54,8 +60,8 @@ object DepsRunner extends WorkerMain[Unit] {
     val usedPaths = Files.readAllLines(namespace.get[File]("used").toPath).asScala.toSet
 
     val remove = if (namespace.getBoolean("check_used") == true) {
-      val whitelist = namespace.getList[String]("whitelist").asScala.map(_.tail)
-      (directLabels -- whitelist).filterNot(labelToPaths(_).exists(usedPaths))
+      val usedWhitelist = namespace.getList[String]("used_whitelist").asScala.map(_.tail)
```

Comment:
Just curious: do you know why we do the `.map(_tail)` here?

Reply:
Hmm... I do not.